html-block-and-inline/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Block and Inline</title>
+  </head>
+  <body>
+    <h1>Blue Americana Snuggie® Blanket</h1>
+    <h2>by SnuggieStore</h2>
+    <p>Price: <s>$19.99</s> $14.99</p>
+    <p>
+      Part of our new Americana collection, this <i>beautiful</i> blue is
```

Comment:
We want to use `em` over `i`. This is because `em` is a semantic element. We will touch on this in Q&A